content/en/workflows/trigger.md

```diff
@@ -45,21 +40,21 @@ To trigger a workflow from a Dashboard, add the **Run Workflow** widget:
 1. From your Dashboard, click **Add Widget**.
 1. Search for `workflows` and add the **Run Workflow** widget.
 1. Under **Select the workflow**, find your workflow in the dropdown menu.
-1. Enter a title for the widget and click **Save**
+1. In the second step, you will be asked to map template variables to the workflow input parameters if any.\
+_This will allow the values of the template variables to be mapped directly to the input parameters when you run the Workflow._
+1. Enter a title for the widget if you want and click **Save**
 
 {{< img src="workflows/trigger-from-dashboard.png" alt="Click Run Workflow to trigger a workflow from Dashboard widget." >}}
```

Comment:
@hestonhoffman please use the [image here]([ttps://a.cl.ly/DOukpRr8](https://www.google.com/url?q=https://a.cl.ly/DOukpRr8&sa=D&source=docs&ust=1676046966244913&usg=AOvVaw14a-3TYHenR3QZ6IjPyFY3))

Reply:
[ttps://a.cl.ly/DOukpRr8](https://www.google.com/url?q=https://a.cl.ly/DOukpRr8&sa=D&source=docs&ust=1676046966244913&usg=AOvVaw14a-3TYHenR3QZ6IjPyFY3)